cc/mcts_node.cc

```diff
@@ -413,7 +415,8 @@ void MctsNode::PruneChildren(Coord c) {
 
 std::array<float, kNumMoves> MctsNode::CalculateChildActionScore() const {
   float to_play = position.to_play() == Color::kBlack ? 1 : -1;
-  float U_scale = kPuct * std::sqrt(std::max<float>(1, N() - 1));
+  float uct = 2 * (std::log((1.0f + N() + kUct_base) / kUct_base) + kUct_init);
+  float U_scale = uct * std::sqrt(std::max<float>(1, N() - 1));
```

Comment:
Consider making adding an inline `U_scale` method to `MctsNode`.

Reply:
Done.